packages/frontend/src/components/common/SpaceActionModal/index.tsx

```diff
@@ -229,7 +229,7 @@ const SpaceActionModal: FC<Omit<ActionModalProps, 'data' | 'isDisabled'>> = ({
             const result = await createMutation({
                 name: state!.name,
                 isPrivate: state!.isPrivate,
-                access: state!.access.map((access) => ({
+                access: state!.access?.map((access) => ({
```

Comment:
is it ok to mutate with undefined? OR would you prefer to default to an empty list?

Reply:
Would prefer to default to empty list! Sounds good @owlas

Reply:
it is ok to mutate with undefined bc of: 
https://github.com/lightdash/lightdash/blob/main/packages/backend/src/services/SpaceService/SpaceService.ts#L121